ROADMAP.md

```diff
@@ -50,3 +50,10 @@ registration way via `registerCandidate` method call is supported via
 Removal of `registerCandidate`–based support of candidate registration will be
 done once `registerCandidate` method is officially deprecated and removed from
 the NeoToken manifest with the subsequent hardfork.
+
+## Replaced math.Max / math.Min with built-in max / min
```

Comment:
This heading is under `Deprecated functionality` section, so here we should just enumerate deprecated things. So it's just something like `math.Max and math.Min interop functions`.